unirec/unirec.c

```diff
@@ -1350,6 +1336,35 @@ int ur_set_from_string(const ur_template_t *tmpl, void *data, ur_field_id_t f_id
    return rv;
 }
 
+uint8_t ur_time_from_string(ur_time_t *ur, const char *str)
```

Comment:
We should check whether *str is not NULL. It would result in a Segmentation fault as strptime does not check it. This should be also tested in tests.